Automattic/jetpack
# Connection: properly add GET-parameters for the `fetchAuthorizationUrl` API call

#### Changes proposed in this Pull Request:
Bug: `fetchAuthorizationUrl` API method (RNA API package) did not include the `no_iframe` parameter in "plain permalinks" mode.
Therefore, site-connected users are unable to connect their user accounts to WP.com if the site has "plain permalinks" enabled.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
_First, reproduce the issue without checking out the branch._
1. Connect Jetpack in site-only mode.
2. Go to "Settings -> Permalinks" and choose "Plain".
3. Go to Jetpack Dashboard and try to connect the user. You should end up on the `jetpack.authorize_iframe` page with minimal UI.
4. Now checkout the branch, rebuild `plugins/jetpack` if needed, and try to connect the user one more time. Confirm that everything went well.
